src/test/java/seedu/address/testutil/TypicalVisits.java

```diff
@@ -101,11 +101,17 @@ public static VisitBook getMoreThanSixtyPercentVisitBook() {
                 SIXTH_VISIT, SEVENTH_VISIT, EIGHTH_VISIT, NINTH_VISIT, TENTH_VISIT, ELEVENTH_VISIT));
     }
 
-    public static List<Index> getLocationsIdsFormVisitsForTest() {
+    public static List<Index> getLocationsIdsFromVisitsForTest() {
         return new ArrayList<>(Arrays.asList(INDEX_SECOND, INDEX_THIRD, INDEX_FIFTH, INDEX_FOURTH));
     }
 
+    /**
+     * Returns a list of Visits where more than 60% of the Visits in this list consist of Person
+     * that is infected
+     */
     public static List<Visit> getMoreThanSixtyPercentInfectedVisits() {
+        // FIRST_VISIT AND SECOND_VISIT are non-infected Visits.
+        // The remaining Visits are infected Visits.
```

Comment:
Suggest putting both comments in 1 line rather than having multiple single line comments.

Reply:
Done.